woocommerce-payments.php

```diff
@@ -287,3 +287,5 @@ function wcpay_get_jetpack_idc_custom_content(): array {
 
 	return $custom_content;
 }
+
+add_filter( 'jetpack_offline_mode', '__return_false' );
```

Comment:
This looks like it may have been left in unintentionally.

Reply:
Fixed with 46b99d1d18edefed06d571559ae8487e2e1e8bf0